src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './NavBar.module.css';
+
+function NavBar() {
+    return (
+        <nav className={styles.navBar}>
+            <ul className={styles.navList}>
+                <li className={styles.navItem}>Home</li>
+                <li className={styles.navItem}>About Us</li>
+                <li className={styles.navItem}>How It Works</li>
+                <li className={styles.navItem}>Contact</li>
+            </ul>
+        </nav>
+    );
+}
```

Comment:
Pay attention that you didn't add this component to the Header component. (I added locally so in the meantime I could give you a CR).